src/blocks/media-block/image-block/CleanImageBlock.tsx

```
@@ -0,0 +1,358 @@
+  const handleError = useCallback((_e: React.SyntheticEvent<HTMLImageElement>) => {
+    console.error('Failed to load image:', mainSrc);
+    
+    // Call the onError callback if provided
+    if (onError) {
+      const error = new Error(`Failed to load image: ${mainSrc}`);
+      onError(error);
+    }
+  }, [mainSrc, onError]);
+
+  // Render error state
+  if (false) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)